.github/workflows/production.yml

```diff
@@ -94,6 +94,16 @@ jobs:
             - name: Building docker image 🐳
               run: docker build -t ${{ secrets.DOCKHUB_ORGANISATION }}/deriv-com:latest -t ${{ secrets.DOCKHUB_ORGANISATION }}/deriv-com:$GIT_TAG_NAME .
 
+            - name: Verify nginx image
+              run: |
+                  nginx_test=`docker run --rm -it --entrypoint bash ${{ secrets.DOCKHUB_ORGANISATION }}/deriv-com:latest-staging -c "nginx -t" | grep -ow "test is successful"`
+                  if [[ $nginx_test == "test is successful" ]];then
+                    echo "Image validation successful"
+                    else
+                    echo "Image validation unsuccessful"
+                    exit 1
+                  fi
```

Comment:
we can use `set -e` to enforce error on exit. since our image `regentmarkets/deriv-com` doesn't have `bash` shell so we can avoid that. testing can be achieved even without introducing if-else block. even using `grep` is not needed

Reply:
Thanks for the suggestions. Changes have been pushed.